---
title: "Analysis of published transciptomic data studies to identify common genes and potential markers of bovine respiratory disease (BRD) and respiratory infection."
author: "Shaun van den Hurk"
format:
  docx:
    toc: false
    number-sections: true
    highlight-style: github
bibliography: ../../assets/dataanalysis-references.bib
csl: ../../assets/american-journal-of-epidemiology.csl
---





This uses MS Word as output format. [See here](https://quarto.org/docs/output-formats/ms-word.html) for more information. You can switch to other formats, like html or pdf. See [the Quarto documentation](https://quarto.org/) for other formats.


```{r, echo=FALSE, message=FALSE}
# load a few R packages
library(here)
library(knitr)
```


**Authors**  

* First Autor$^{1,2,*}$ (ORCID: 0000-0000-1234-5678) 
* Second Author$^{3, *}$
* Third Autor$^{3, \dagger}$
* Last Author$^{1, 2, \land}$ 

**Author affiliations**  

1. College of Public Health, University of Georgia, Athens, GA, USA.
2. Another Affiliation, Athens, GA, USA.
3. Yet another Affiliation.

$*$ These authors contributed equally to this work.

$\land$ Corresponding author: some@email.com

$\dagger$ Disclaimer: The opinions expressed in this article are the
author's own and don't reflect their employer.


{{< pagebreak >}}



# Summary/Abstract
_Write a summary of your project._
Respiratory disease is an important condition that affects the health, welfare and production of cattle. Respiratory disease in cattle is often attributed to the Bovine Respiratory Disease Complex (BRDC) and can be caused by a number of different pathogens. Currently there are over 9 different viruses and bacteria that are implicated in the BRDC, with the list growing, and potentiated by factors such as the environment and management factors.

The proposed project is a type of meta-analysis of different published datasets of transcriptomic data as a means to identify some potential biomarkers (such as miRNA) that might be differentially expressed with respiratory infection in bovines. The focus will be on the gene pathways that appear to be differentially expressed. The aim is to identify if there are any associations between infection (broadly and with specific pathogens) and these transcripts, which then could be used as a predictor of disease. 




{{< pagebreak >}}


# Introduction 

## General Background Information
_Provide enough background on your topic that others can understand the why and how of your analysis_ 

## Description of data and data source
_Describe what the data is, what it contains, where it is from, etc. Eventually this might be part of a methods section._
The data to be used is from published studies and online data found on the NCBI Gene Expression Omnibus (GEO) database. A search was performed on this database to look for data related to bovine respiratory disease as well as transcriptome, and/or protein and/or genes. There were between 10 than 22 results search results (depending on the search) on the database, and from these projects with published data, relevant projects are being identified. The studies are only included if they have retrievable data uploaded, either on the GEO database or the corresponding published article (including supplemental data).


Currently, five potential suitable sources have been identified, one such source is listed on GEO as "GSE199108", with the published article citation "O'Donoghue, S., Earley, B., Johnston, D., Mccabe, M.S., KIM, J., Taylor, J.F., Duffy, C., Lemon, K., Mcmenamy, M., Cosby, S.L. and Morris, D.W., Whole blood transcriptome analysis in dairy calves experimentally challenged with Bovine Herpesvirus 1 (BoHV-1) and comparison to a Bovine Respiratory Syncytial Virus (BRSV) challenge. Frontiers in Genetics, 14, p.171.".
From the mentioned resources, the data with the gene counts and other raw data will be retrieved and these Excel and Text files will provide the raw data. 
The researchers from these studies published the raw data and the scientific journal article, but not their analysis or their workflow for this. Additionally, each of these studies was performed with direct sampling, and I am not familiar with a study that has combined all of the data in a meta-analysis with collation of the data for a deeper analysis of these transcriptome profiles in cattle.



## Questions/Hypotheses to be addressed
_State the research questions you plan to answer with this analysis._

- Are there differences in the expression of genes and proteins between healthy and infected cattle that can be identified through transciptomic analysis? Can these genes be identied in this way.

- Are there any genes or biomarkers that are associated with infection with particular pathogens?

- Are there any common genes or biomarkers identified that are associated with infection with multiple different pathogens?


Thus the outcomes of interest are primarily the levels of expression of these targets and the disease status of the animals sampled.




To cite other work (important everywhere, but likely happens first in introduction), make sure your references are in the bibtex file specified in the YAML header above and have the right bibtex key. Then you can include like this:

Examples of reproducible research projects can for instance be found in [@mckay2020; @mckay2020a].



{{< pagebreak >}}


# Methods 

_Describe your methods. That should describe the data, the cleaning processes, and the analysis approaches. You might want to provide a shorter description here and all the details in the supplement._

The raw data has samples from healthy and experimentally infected animals which will be compared to get an indication of differential expression of certain genes between these groups. This will serve as the basis for the analysis as the focus will be on the subset of differentially expressed genes.
The raw data from the different sources will be cleaned up in R and analysed before they are combined in a single analysis to look for common features and to compare the expression profiles. Heatmaps will be helpful to help identify and visualise the differential expression of genes, and an approach might be to sample the top 10 genes with high and low levels of expression. 
Statistical analysis will focus largely on the associations between the factors, for example a t-test, and regression models might be considered too. We will look for associations between gene expression and infection status (heath vs infected).

It is expected that there will be some obvious differences in these levels of expression that will be asociated with the disease status and that many of these will be common between the different respiratory pathogens.

If potential genes or biomarkers are identified they will be run through online sources and databases (including tools such as the PubChem online tool) to identify the common names (or symbol) and proposed functions of these genes.



## Schematic of workflow

Sometimes you might want to show a schematic diagram/figure that was not created with code (if you can do it with code, do it). @fig-schematic is an example of some - completely random/unrelated - schematic that was generated with Biorender.
We store those figures in the `assets` folder.

```{r}
#| label: fig-schematic
#| fig-cap: "A figure that is manually generated and shows some overview/schematic. This has nothing to do with the data, it's just a random one from one of our projects I found and placed here."
#| echo: FALSE
knitr::include_graphics(here("assets","antigen-recognition.png"))
```




## Data aquisition
_As applicable, explain where and how you got the data. If you directly import the data from an online source, you can combine this section with the next._


## Data import and cleaning
_Write code that reads in the file and cleans it so it's ready for analysis. Since this will be fairly long code for most datasets, it might be a good idea to have it in one or several R scripts. If that is the case, explain here briefly what kind of cleaning/processing you do, and provide more details and well documented code somewhere (e.g. as supplement in a paper). All materials, including files that contain code, should be commented well so everyone can follow along._


## Statistical analysis
_Explain anything related to your statistical analyses._


{{< pagebreak >}}


# Results

## Exploratory/Descriptive analysis

_Use a combination of text/tables/figures to explore and describe your data. Show the most important descriptive results here. Additional ones should go in the supplement. Even more can be in the R and Quarto files that are part of your project._


@tbl-summarytable shows a summary of the data.

Note the loading of the data providing a **relative** path using the `../../` notation. (Two dots means a folder up). You never want to specify an **absolute** path like `C:\ahandel\myproject\results\` because if you share this with someone, it won't work for them since they don't have that path. You can also use the `here` R package to create paths. See examples of that below. I generally recommend the `here` package.

```{r}
#| label: tbl-summarytable
#| tbl-cap: "Data summary table."
#| echo: FALSE
resulttable=readRDS("../../results/tables/summarytable.rds")
knitr::kable(resulttable)
```



## Basic statistical analysis

_To get some further insight into your data, if reasonable you could compute simple statistics (e.g. simple models with 1 predictor) to look for associations between your outcome(s) and each individual predictor variable. Though note that unless you pre-specified the outcome and main exposure, any "p<0.05 means statistical significance" interpretation is not valid._


@fig-result shows a scatterplot figure produced by one of the R scripts.

```{r}
#| label: fig-result
#| fig-cap: "Height and weight stratified by gender."
#| echo: FALSE
knitr::include_graphics(here("results","figures","height-weight-stratified.png"))
```


## Full analysis

_Use one or several suitable statistical/machine learning methods to analyze your data and to produce meaningful figures, tables, etc. This might again be code that is best placed in one or several separate R scripts that need to be well documented. You want the code to produce figures and data ready for display as tables, and save those. Then you load them here._

Example @tbl-resulttable2 shows a summary of a linear model fit.

```{r}
#| label: tbl-resulttable2
#| tbl-cap: "Linear model fit table."
#| echo: FALSE
resulttable2 = readRDS(here("results","tables","resulttable2.rds"))
knitr::kable(resulttable2)
```


{{< pagebreak >}}


# Discussion

## Summary and Interpretation
_Summarize what you did, what you found and what it means._

## Strengths and Limitations
_Discuss what you perceive as strengths and limitations of your analysis._

## Conclusions
_What are the main take-home messages?_

_Include citations in your Rmd file using bibtex, the list of references will automatically be placed at the end_

This paper [@leek2015] discusses types of analyses. 

These papers [@mckay2020; @mckay2020a] are good examples of papers published using a fully reproducible setup similar to the one shown in this template. 

Note that this cited reference will show up at the end of the document, the reference formatting is determined by the CSL file specified in the YAML header. Many more style files for almost any journal [are available](https://www.zotero.org/styles). You also specify the location of your bibtex reference file in the YAML. You can call your reference file anything you like.


{{< pagebreak >}}

# References



